compiler/damlc/tests/src/DA/Test/ScriptService.hs

```diff
@@ -919,25 +919,17 @@ main =
                     , "  p2 <- allocateParty \"p2\""
                     , "  cid <- submit p1 $ createCmd (LocalKeyVisibility p1 p2)"
                     , "  submit p2 $ exerciseCmd cid LocalFetch"
-                    , "localLookupFetchMustFail = do"
```

Comment:
lookup and fetch testing is done above. A lookup and fetch test now always fails on the fetch as contract has been consumed.